src/lib/hooks/usePublicAccounts/index.ts

```diff
@@ -39,34 +35,28 @@ export interface ParsedAccountType {
   categories: definitions["categories"]["id"][];
   sensorsCount: number;
   recordsCount: number;
-  sensors: ParsedSensorType[];
+  sensors: definitions["extended_user_profiles"]["sensors"];
 }
 
 export const mapPublicAccount = ({
-  sensors,
   ...user
-}: AccountQueryResponseType): ParsedAccountType => ({
-  id: user.id,
-  username: user.name || "anonymous",
-  displayName: user.display_name || user.name || "Anonymous",
-  createdAt: user.created_at,
-  link: user.url,
+}: definitions["extended_user_profiles"]): ParsedAccountType => ({
+  id: user.id || "",
```

Comment:
All these fallbacks are necessary because the generated types are telling me that these fields may be `null`. I know that in reality this won't really be the case, but I didn't want to ignore the types.

Perhaps they are nullable because the DB view treats columns differently than "real" tables?

Reply:
Might be the case. I take a look into pg views. Maybe we can fix this

Reply:
Hm. A view will not inherit the constrains of the source tables. ;(